internal/wazeroir/operations.go

```diff
@@ -797,49 +717,57 @@ func NewOperationBuiltinFunctionCheckExitCode() UnionOperation {
 	return UnionOperation{OpKind: OperationKindBuiltinFunctionCheckExitCode}
 }
 
-// Label is the label of each block in wazeroir where "block" consists of multiple operations,
-// and must end with branching operations (e.g. OperationBr or OperationBrIf).
-type Label struct {
+// label is the label of each block in wazeroir where "block" consists of multiple operations,
+// and must end with branching operations (e.g. NewOperationBr or NewOperationBrIf).
+type label struct {
```

Comment:
curious if we need this type in the first place. Looks like we could remove this and name `LabelID` to `Label`.

Reply:
removed :)